src/LibP4est_v2.jl

```diff
@@ -7080,9 +7080,9 @@ function Base.getproperty(x::Ptr{p6est_quadrant_data}, f::Symbol)
     f === :user_long && return Ptr{Clong}(x + 0)
     f === :user_int && return Ptr{Cint}(x + 0)
     f === :which_tree && return Ptr{p4est_topidx_t}(x + 0)
-    f === :piggy1 && return Ptr{__JL_Ctag_317}(x + 0)
-    f === :piggy2 && return Ptr{__JL_Ctag_318}(x + 0)
-    f === :piggy3 && return Ptr{__JL_Ctag_319}(x + 0)
+    f === :piggy1 && return Ptr{__JL_Ctag_320}(x + 0)
+    f === :piggy2 && return Ptr{__JL_Ctag_315}(x + 0)
+    f === :piggy3 && return Ptr{__JL_Ctag_316}(x + 0)
```

Comment:
Are these `__JL_Ctag_*` with different numbers basically the same structs?

Reply:
Yes, they are the same. I don't know why we need those twice each.

Reply:
Thanks!